apis/sample/v1alpha1/doc.go

```diff
@@ -14,8 +14,4 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package v1alpha1 contains the v1alpha1 group Sample resources of the Template provider.
-// +kubebuilder:object:generate=true
-// +groupName=sample.template.crossplane.io
-// +versionName=v1alpha1
```

Comment:
@muvaf do we still need a `doc.go` with all docstrings removed?

Reply:
I prefer to keep a docstring for each `package` stanza, even if they are pretty boilerplate like this one.

Reply:
Oh - I see it's still there, just in a different file. Does pkg.go.dev really care which file specifically the docstring is in?

Reply:
> Does pkg.go.dev really care which file specifically the docstring is in?

I'm not 100% sure but that's what I got from their documentation. The comment in `doc.go` is used as the package overview. See https://go.dev/blog/godoc